src/ValidatorChain.php

```diff
@@ -69,11 +81,9 @@ private function attachValidator($event, $callback, $priority)
             array_unshift($callback, $test);
         }
         if ($context instanceof ValidatorInterface) {
```

Comment:
Because the validator chain is specific to this lib, why not change this method signature to `attachValidator(Event $event, ValidatorInterface $validator, int $priority)` for better type inference? _(Possibly better in another patch)_